localization/languages/en-US.json

```diff
@@ -23,6 +23,7 @@
     "cut": "Cut",
     "copy": "Copy", //this is a verb (as in "copy the currently-selected text")
     "paste": "Paste",
+    "pasteAndMatchStyle": "Paste and Match Style",
```

Comment:
Can you also add this line to the other language files, so the translators know to add it?

```
    "pasteAndMatchStyle": null, //missing translation
```